appium/webdriver/extensions/android/nativekey.py

```diff
@@ -0,0 +1,1057 @@
+    # Menu (0x11) of CEC User Control Code.
+    _TV_MEDIA_CONTEXT_MENU = 257
+
+    # Key code constant: Timer programming key.
+    # Goes to the timer recording menu. Corresponds to Timer Programming (0x54) of
+    # CEC User Control Code.
+    TV_TIMER_PROGRAMMING = 258
+
+    # Key code constant: Help key.
+    HELP = 259
+
+    # Key code constant: Navigate to previous key.
+    # Goes backward by one item in an ordered collection of items.
+    NAVIGATE_PREVIOUS = 260
+
+    # Key code constant: Navigate to next key.
+    # Advances to the next item in an ordered collection of items.
+    NAVIGATE_NEXT = 261
+
+    # Key code constant: Navigate in key.
+    # Activates the item that currently has focus or expands to the next level of a navigation
+    # hierarchy.
+    NAVIGATE_IN = 262
+
+    # Key code constant: Navigate out key.
+    # Backs out one level of a navigation hierarchy or collapses the item that currently has
+    # focus.
+    NAVIGATE_OUT = 263
+
+    # Key code constant: Primary stem key for Wear.
+    # Main power/reset button on watch.
+    STEM_PRIMARY = 264
+
+    # Key code constant: Generic stem key 1 for Wear.
+    STEM_1 = 265
+
+    # Key code constant: Generic stem key 2 for Wear.
+    STEM_2 = 266
+
+    # Key code constant: Generic stem key 3 for Wear.
+    STEM_3 = 267
+
+    # Key code constant: Directional Pad Up-Left.
+    DPAD_UP_LEFT = 268
+
+    # Key code constant: Directional Pad Down-Left.
+    DPAD_DOWN_LEFT = 269
+
+    # Key code constant: Directional Pad Up-Right.
+    DPAD_UP_RIGHT = 270
+
+    # Key code constant: Directional Pad Down-Right.
+    DPAD_DOWN_RIGHT = 271
+
+    # Key code constant: Skip forward media key.
+    MEDIA_SKIP_FORWARD = 272
+
+    # Key code constant: Skip backward media key.
+    MEDIA_SKIP_BACKWARD = 273
+
+    # Key code constant: Step forward media key.
+    # Steps media forward, one frame at a time.
+    MEDIA_STEP_FORWARD = 274
+
+    # Key code constant: Step backward media key.
+    # Steps media backward, one frame at a time.
+    MEDIA_STEP_BACKWARD = 275
+
+    # Key code constant: put device to sleep unless a wakelock is held.
+    SOFT_SLEEP = 276
+
+    # Key code constant: Cut key.
+    CUT = 277
+
+    # Key code constant: Copy key.
+    COPY = 278
+
+    gamepad_buttons = [BUTTON_A, BUTTON_B, BUTTON_C, BUTTON_X,
+                       BUTTON_Y, BUTTON_Z, BUTTON_L1, BUTTON_R1,
+                       BUTTON_L2, BUTTON_R2, BUTTON_THUMBL, BUTTON_THUMBR,
+                       BUTTON_START, BUTTON_SELECT, BUTTON_MODE, BUTTON_1,
+                       BUTTON_2, BUTTON_3, BUTTON_4, BUTTON_5, BUTTON_6,
+                       BUTTON_7, BUTTON_8, BUTTON_9, BUTTON_10,
+                       BUTTON_11, BUTTON_12, BUTTON_13, BUTTON_14,
+                       BUTTON_15, BUTTON_16]
+
+    @staticmethod
+    def is_gamepad_button(code):
+        """Returns true if the specified nativekey is a gamepad button."""
```

Comment:
Could you use the same sentence format for  similar functions? 
I prefer this one.